install_config/syncing_groups_with_ldap.adoc

```diff
@@ -1049,6 +1049,14 @@ The object specification for the configuration file is below.  Note that the dif
 objects have different fields.  For example, xref:sync-ldap-v1-activedirectoryconfig[v1.ActiveDirectoryConfig] has no `groupsQuery`
 field whereas xref:sync-ldap-v1-rfc2307config[v1.RFC2307Config] and xref:sync-ldap-v1-augmentedactivedirectoryconfig[v1.AugmentedActiveDirectoryConfig] both do.
 
+[IMPORTANT]
+====
+There is no support for binary attributes. All attribute data coming from the
+LDAP server must be in the format of a UTF-8 encoded string. For example, never
+use a binary attribute as an ID attribute (`objectGUID`). String attributes must
```

Comment:
never use a binary attribute such as `objectGUID` as an ID attribute. String attributes such as `sAMAccountName` or `userPrincipalName` must be used instead.